CONTRIBUTING.rst

```diff
@@ -84,7 +84,7 @@ Ready to contribute? Here's how to set up `nxviz` for local development.
 
    To get flake8 and tox, just pip install them into your virtualenv.
 
-6. Commit your changes and push your branch to GitHub::
+6. Commit your changes and push your branch to GitHub:
```

Comment:
The double colon (`::`) is Sphinx syntax to indicate that the coming block is to be formatted like code. It's a minor thing, so I'll change this back on my own, no need to worry about this :smile:.